cmd/ktor/main.go

```diff
@@ -73,15 +74,40 @@ func main() {
 			},
 		}
 
-		projectName := utils.CleanProjectName(args.CommandArgs[0])
-		projectDir, err := filepath.Abs(projectName)
+		if len(args.CommandArgs) > 0 {
+			projectName := utils.CleanProjectName(args.CommandArgs[0])
+			projectDir, err := filepath.Abs(projectName)
 
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "Unable to determine project directory for project %s\n", projectName)
```

Comment:
Please consider extracting error messages to a separate file so we can think about proofreading and localization

Reply:
I am going to do that in a separate PR (https://youtrack.jetbrains.com/issue/KTOR-7764).